database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -20,7 +17,7 @@ public function up(): void
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
             $table->rememberToken();
-            $table->string('language')->default(Language::English->value);
+            $table->string('language')->default('en');
```

Comment:
It is rather convenient using literal values on non-app code! (And makes me not hate the `->value` access)